meson.build

```diff
@@ -0,0 +1,66 @@
+project('libva-utils', 'c', 'cpp',
+        version : '2.1.1',
+        default_options : [
+          'warning_level=2',
+          'c_std=gnu99',
+          'default_library=static',
+        ],
+        license : 'MIT',
+        meson_version : '>= 0.44.0')
+
+add_project_arguments([ '-Wno-unused-parameter',
+                        '-Wno-sign-compare' ],
+                      language: ['c', 'cpp'])
+
+c = meson.get_compiler('c')
+
+libva_dep = dependency('libva', version: '>= 1.1.0')
+
+# DRM
+use_drm = get_option('drm') == 'true'
+drm_deps = [
+  dependency('libdrm', version: '>= 2.4', required: use_drm),
+  dependency('libva-drm', required: use_drm),
+]
+use_drm = true
+foreach d : drm_deps
+  if not d.found()
+    use_drm = false
+  endif
+endforeach
+
+# X11
+use_x11 = get_option('x11') == 'true'
+x11_deps = [
+  dependency('x11', required: use_x11),
+  dependency('xext', required: use_x11),
+  dependency('xfixes', required: use_x11),
+  dependency('libva-x11', required: use_x11),
+]
+use_x11 = true
+foreach d : x11_deps
+  if not d.found()
+    use_x11 = false
+  endif
+endforeach
+
+# WAYLAND
+use_wayland = get_option('wayland') == 'true'
+wayland_deps = [
+  dependency('wayland-client', version: '>= 1.0.0', required: use_wayland),
+  dependency('libva-wayland', required: use_wayland),
+]
+use_wayland = true
+foreach d : x11_deps
+  if not d.found()
+    use_wayland = false
+  endif
+endforeach
+
```

Comment:
If user specifies any of `-Ddrm=false`, `-Dx11=false`, `-Dwayland=false`, they are not honored here.  That is, if user has all the dependencies installed for these features, they still may want to disable them here.

Reply:
Hmm... Right, need to switch the default value to != false.